packages/compas-save-plugin/src/helpers/foundation.ts

```diff
@@ -0,0 +1,61 @@
+import { LitElement } from "lit-element";
+import {
+  getTypeFromDocName,
+  CompasSclDataService,
+  NOT_FOUND_ERROR,
+} from "@com-pas/compas-open-core";
+
+/** Constructor type for defining `LitElement` mixins. */
+export type LitElementConstructor = new (...args: any[]) => LitElement;
+
+/** The type returned by `MyMixin(...)` is `Mixin<typeof MyMixin>`. */
+export type Mixin<T extends (...args: any[]) => any> = InstanceType<
+  ReturnType<T>
+>;
+
+export function callService(docType: string, docId: string): Promise<Document> {
```

Comment:
Can this function be renamed or removed? looks like it's just a passthrough to the CompasSclDataService